generator/src/html/script/Main.hx

```diff
@@ -18,9 +19,34 @@ class Main {
 		// parse and locate myselft
 		assert(tocData != null, "toc data is missing");
 		var toc = J(JQuery.parseHTML(tocData));
-		assert(document.URL.startsWith(document.baseURI));
-		var myUrl = document.URL.replace(document.baseURI, "").replace(window.location.hash, "");
-		if (myUrl == "")
+        
+        var baseURI = null;// document.baseURI ;
+        //FIXME IE workaround for the lack of document.baseURI 
+        if (baseURI == null) {
```

Comment:
@jonasmalacofilho: As the workaround will break the moment somebody gets near it... I think we should be spared the shame of breaking everything on account of a patch to IE. Let it be for now... and I will make a new pull request to have the independent property in base as you proposed  `<base id="docbase" href="?" x-rel-dir="?">` on the other channel (I will make an issue about it, to the case I stop working on it in the next few days)